auth/initialConf.py

```diff
@@ -164,7 +165,8 @@ def add_permissions_group():
                 'all_mashup',
                 'ro_alarms',
```

Comment:
Does it make sense the user profile be able to read the alarms of the system?

Reply:
Well, system alarms may not be of interest for a normal user, but if these alarms include notifications of, say, iotagents indicating that messages from a particular device belonging to that user could not be processed, that might be useful.